### PR TITLE
documentation/614

### LINKED ISSUE #614: Refine Sprint Process Checklists
Follow-up to openscope/openscope#418 (feature #511). Some things have gone slightly awry with the checklist that weren't expected. Will need a closer look to get everything going properly.

- [x] Non-continuous numbering
- [x] `Deploy v1.2.3` sha on `master` must be merged back into `develop` at some point (null diff).
    - At `ARCH - Prepare CHANGELOG`? At end of release procedure?
- [x] Fix invalid tag pushing command to `git push origin tagName`
- [x] allow expansion of branch naming conventions

img_20170810_125225

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #614.

Update git strategy document

### CODE REVIEW

**Comment by erikquinn** [comment]
As amended, everything went as it should this time around.

File: documentation/development-processes-checklists.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-    - includes GitHub labels `bugfix` (regular bugfixes) and `hotfix` (critical bug fix pushed directly to production)
-- `feature`: for changes that add or notably improve an existing feature
-    - includes GitHub labels `airport work` (new airports only), `enhancement`, and of course `feature`
-- `refactor`: for changes that neither add features nor fix bugs, like code reorganization
-    - includes GitHub labels `airport work` (all airport updates), `devops`, `documentation`, and of course `refactor`
+Each issue and corresponding pull request will be placed in one of the following categories:
+- `feature`: addition of a _completely new feature_
+- `enhancement`: notable _improvement or expansion upon an existing feature_
+- `refactor`: code changes that _neither fix a defect nor add functionality_
+- `bugfix` and `hotfix`: fix a _defect of the simulator itself_
```
Comment (n8rzz) [inline documentation/development-processes-checklists.md]: what makes a `bugfix` a `bugfix` and a `hotfix` a `hotfix`?  each should have their own line with description.

Comment (erikquinn) [reply documentation/development-processes-checklists.md]: you're right. not sure why I was trying to avoid it. Done!

**Comment by erikquinn** [comment]
@n8rzz bump

**Comment by n8rzz** [comment]
@erikquinn obviously, be sure to address the conflict before merging.

**Pull request merged**